include/z64.h

```diff
@@ -399,10 +399,14 @@ typedef struct {
     /* 0x01FC */ s16    unk_1FC;
     /* 0x01FE */ s16    unk_1FE;
     /* 0x0200 */ s16    unk_200;
-    /* 0x0202 */ s16    unk_202[3];
-    /* 0x0208 */ s16    unk_208[3];
-    /* 0x020E */ s16    unk_20E[6];
-    /* 0x021A */ s16    unk_21A[6];
+    /* 0x0202 */ s16    beatingHeartPrim[3];
+    /* 0x0208 */ s16    beatingHeartEnv[3];
+    /* 0x020E */ s16    healthPrimR[2];
+    /* 0x0212 */ s16    healthPrimG[2];
+    /* 0x0216 */ s16    healthPrimB[2];
+    /* 0x021A */ s16    healthEnvR[2];
+    /* 0x021E */ s16    healthEnvG[2];
+    /* 0x0222 */ s16    healthEnvB[2];
```

Comment:
Thoughts on using the term heart here, since that's what you went with for all other colors? (as far as I can tell)

Reply:
Ended up switching a lot of things over to health so it can probably stay as is

Reply:
That's also fine I guess, but now it's still not consistent with "beating heart" names, so I'm not sure

Reply:
yeah a bit unsure of what to name that, I was thinking trailingHeart but that also has "heart" in it. maybe trailingHealth?

Reply:
if hearts instead of health makes more sense, it might be better to go in that direction for everything

Reply:
I also think it could make sense to go with hearts for all these color names

Reply:
alright when i get some time ill make that change then